main.css

```diff
@@ -123,6 +131,28 @@ h3.headline {
 }
 
 
+/* Banner ---------------------------------------- */
+
+.banner {
+  padding: 1em;
+  width: 360px;
```

Comment:
I don't think this is desirable. Let's come up with a better way to handle this.